### ISSUE TITLE
Scheduled tasks grouping

### ISSUE THREAD

**Issue by loziniak** [jenspfahl/EverydayTasks#13]
It'd be nice to have sheduled tasks grouped, eg. like this:

```text
* overdue task
------------ today ------------- // from now till midnight (or configurable, eg. 4am)
* due task
* task in 2 hours
----------- tomorrow ----------- // from midnight/4am till 24 hours later
* task in 20 hours
---------- this week ----------- // till sunday 4am
* task on thursday
* task on friday
---------- next week ----------- // from sunday 4am till next sunday 4am
* ....
----- this month (october) ----- // from next sunday 4am till 4am nov 1st
* ....
* ....
----------- november ----------- // from 4am nov 1st till 4am dec 1st
* ....
---------- this year ----------- // from 4am dec 1st till 4am jan 1st 2023
* ....
------------- 2023 ------------- // 4am jan 1st 2023 till 4am jan 1st 2024
* ....
* ....
* ....
------------ later ------------- // everything after 4am jan 1st 2024
* ....
```

**Comment by jenspfahl** [comment]
Good idea. Currently you can sort by remaining time or progress. So remaining time is what comes close to your suggestion.

**Comment by jenspfahl** [comment]
Shipped with v1.6.0, but with a different grouping.
